users/config.rst

```diff
@@ -475,13 +531,13 @@ compression
     never
         Disable all compression.
 
-introducer
+certName
+    The device certificate common name, if it is not the default "syncthing".
+
+:ref:`introducer <introducer>`
```

Comment:
Suggested link text: "Refer to the **Introducer Configuation** section for an overall explanation of this mechanism." (bold = link)